### DISCUSSION TITLE
Satellite icon speed

### DISCUSSION THREAD

**Discussion by clanker24** [Stellarium/stellarium#3862] [General]
Perhaps someone here has investigated this. When one measures the orbital velocity of satellites as shown in Stellarium, the velocity works out to be much faster than an orbiting satellite should be moving. If we assume that a satellite is in low earth orbit (200-800 km), it's orbital velocity is about (7.5 km/sec + or -). It makes one complete orbit of the earth in about 95 minutes. It is easier to think of this in terms of degrees/second (360 degrees/95 minutes x 60 sec/min) = .0632 degrees/sec (15.8 sec/ degree of arc).

Take any satellite shown in Stellarium at random and measure the time it takes for the satellite to move from horizon to horizon (pick an satellite orbit that goes overhead). Then use Stellarium angle measurement tool and measure the length of the arc. Then calculate its movement in degrees of arc per second. You will find that number to be about 4 times higher than it should be. For example, I chose Delfi-N3xt, a Dutch satellite with an orbital period of 97.3 minutes. I timed its movement across the Stellarium sky through an arc of 168.6 degrees. It took 651 seconds, or 0.259 degrees/sec. Since I know from published data that its period is 97.3 minutes, it should orbit at 360 degree/ 5838 sec or 0.0617 degrees/sec. The Stellarium icon's movement is more than 4 times faster than the satellite is actually moving.

Can anyone here explain this?

**Comment by gzotti** [comment]
You are not standing in Earth's centre but on its surface, 6378km away or so. The satellite zooms by just 500km over you. Geocentric angular velocity is not applicable here.

**Comment by 10110111** [comment]
The orbit is focused in the center of mass of the Earth (neglecting the satellite's mass), while you're looking at the sky from Earth's surface. That's about 6400 km of difference in distance to the satellite, while satellites' altitudes above the surface are generally an order of magnitude smaller. So surely the angle a satellite covers in the observer's sky per unit of time is much larger than the angle (so called _true anomaly_) it passes along its orbit with respect to the orbit's focus.
